index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf8" />
     <title>Quiz</title>
 
-    <link href="./public/style.css" rel="stylesheet" />
+    <link href="./public/style.scss" rel="stylesheet" />
```

Comment:
This change is already in another PR, which means you created this branch `2_2` when you were in `2_1` not in `main`